paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -31,7 +31,22 @@ class PaparazziPluginTest {
 
     assertThat(result.task(":preparePaparazziDebugResources")).isNull()
     assertThat(result.output).contains(
-        "The Android Gradle library plugin must be applied before the Paparazzi plugin."
+        "The Android Gradle library plugin must be applied together with Paparazzi plugin."
```

Comment:
```suggestion
        "The Android Gradle library plugin must be applied for Paparazzi to be configured."
```

Reply:
This suggestion still kind of implies order.